Specs/Scene/PointCloud3DTileContentSpec.js

```diff
@@ -16,7 +16,7 @@ defineSuite([
         'Specs/Cesium3DTilesTester',
         'Specs/createScene',
         'ThirdParty/when'
-    ], function(
+    ], 'Scene/PointCloud3DTileContent', function(
```

Comment:
This doesn't look right to me.  Why is this outside of the array?

Reply:
Oh, I see, we're not actually using the `PointCloud3DTileContent` class directly in these specs

Reply:
We may have added this to jasmine, but adding it after the array makes it the suite name. If this isn't added, the tests are run under the first require.
See : https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/Scene/GeometryRenderingSpec.js#L41